services/gps2tsdb/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.1'
+services:
+  # docker run -it -v /var/run/dbus:/var/run/dbus --rm --name my-running-app my-python-app
+  gps2tsdb:
+    build:
+      # Cannot use `./` due to this issue https://github.com/docker/compose/issues/3530#issuecomment-222490501
+      context: ../gps2tsdb
+    restart: always
+    environment:
+      db_user: "avena"
+      db_host: "postgres"
+      db_database: "avena"
+      db_password: "password"
+      db_port: "5432"
+    depends_on:
+      - postgres
```

Comment:
The postgres/timescaledb docker container. I suppose this container could be run independently for testing purposes, however I am unsure how useful it will be

Reply:
I suppose my concern is that there is no way to know where to get that `postgres` service (and in this case it really is tsdb) because it is defined in a different docker-compose file that I have to remember to include. 

I am a bit confused on the point of these docker-compose files? A compose file is all about deploying a system somewhere. It is very unlikely _this_ compose file will end up on an Avena device.  An Avena device will need one customized to its needs. That why I assumed this compose file is about testing. Maybe I have misunderstood the goal?

Note -- I'm fine with bringing all this in for now as we work through what we want. This just seems like a nice moment to discuss what is it that we want.

Reply:
Resolving to keep discussion here: https://github.com/OATS-Group/isoblue-avena/pull/45#discussion_r461692316